src/foam/nanos/auth/User.js

```diff
@@ -347,6 +349,21 @@ foam.CLASS({
       networkTransient: true,
       section: 'administrative'
     },
+    {
+      name: 'passwordHistory',
+      class: 'FObjectArray',
+      of: 'foam.nanos.auth.PriorPassword',
+      javaFactory: `
+        foam.nanos.auth.PriorPassword[] priorPasswords = new foam.nanos.auth.PriorPassword[1];
+        priorPasswords[0] = new foam.nanos.auth.PriorPassword();
+        priorPasswords[0].setPassword(this.getPassword());
+        priorPasswords[0].setTimeStamp(new Date());
+        return priorPasswords;
+      `,
+      hidden: true,
+      networkTransient: true,
+      section: 'administrative'
```

Comment:
This way this property won't be editable in the GUI. We don't want people changing history.

```suggestion
      section: 'administrative',
      createMode: 'HIDDEN',
      updateMode: 'RO'
```